### PR TITLE
Italian translation (main strings)

### PR DESCRIPTION
**Pull request by shmykelsa**

### CODE REVIEW

**Comment by naman14** [comment]
@shmykelsa can you combine this and #49 into a single PR. you can add commits to existing PRs

**Comment by shmykelsa** [comment]
I don't know how to do that

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    <string name="dark">Scuro</string>
+    <string name="black">Nero</string>
+
+    <string name="settings">Impostazioni</string>
+    <string name="choose_start_page">Scegli la pagina di iniziale all'avvio.</string>
+    <string name="start_page">Pagina iniziale</string>
+    <string name="choose_theme">Seleziona il tema da usare per l'app</string>
+    <string name="theme">Tema</string>
+
+    <string name="load_artist_image">Immagini dell'artista Last.fm</string>
```
Comment (afollestad) [inline app/src/main/res/values-it/strings.xml]: Single quotes (') need to be preceded by a forward slash (\)

Comment (afollestad) [reply app/src/main/res/values-it/strings.xml]: (that's why the build is failing here)

**Pull request closed without merging**